lib/Partitioner/Partitioner.cpp

```diff
@@ -534,6 +536,56 @@ void Partitioner::doPartitioning(Function *F, NodeToFunctionMap &mapping) {
   }
 }
 
+/// Get the partitions of a function \p funcName based on a config file with \p
+/// filename. It is a text file with the format:
+///
+/// # node_name_1 node_name_2 ...node_name_n # node_name_x1 node_name_x2 ...
```

Comment:
It is useful if one could also have comments in the config files as they allow for providing some explanations. Thus it would be nice if you could allow for some kind of comments. Usually comments start with either `//` or `#` in many languages. Therefore your choice of `#` for a partition indicator is probably not the best one ;-)

Reply:
I see. We can start a partition with a number, since in Glow, a valid name should start with alphabet letters.

Reply:
I'm not sure about using a number. May be something like braces? E.g. `[node1 node2 node3]` or `(node1 node2 node3])`. Or `-> node1 node2 node3`...